mm2src/coins/lp_coins.rs

```diff
@@ -2707,23 +2739,103 @@ impl Default for PrivKeyActivationPolicy {
     fn default() -> Self { PrivKeyActivationPolicy::ContextPrivKey }
 }
 
-#[derive(Debug)]
+#[derive(Clone, Debug)]
 pub enum PrivKeyPolicy<T> {
-    KeyPair(T),
+    Iguana(T),
+    HDWallet {
+        /// Derivation path of the coin.
+        /// This derivation path consists of `purpose` and `coin_type` only
+        /// where the full `BIP44` address has the following structure:
```

Comment:
I will write the full derivation path `m/purpose'/coin_type/account'/change/address_index` since it states `This derivation path consists of purpose and coin_type only`

Reply:
Done